test/ParseOutputs/test_parse_outputs.py

```diff
@@ -0,0 +1,46 @@
+import filecmp
+import os
+import shutil
+from pathlib import Path
+
+import yaml
+
+from spras import runner
+from spras.domino import DOMINO, post_domino_id_transform, pre_domino_id_transform
+
+INDIR = "test/ParseOutputs/input/"
+OUTDIR = "test/ParseOutputs/output/"
+EXPDIR = "test/ParseOutputs/expected/"
+
+# for domino
+OUT_FILE_PARSE = OUTDIR+'/domino-parse-output.txt'
+OUT_FILE_PARSE_EXP = EXPDIR + '/domino-parse-output.txt'
+
+# domino is seperate function
+algorithms = ['mincostflow', 'meo', 'omicsintegrator1', "omicsintegrator2", "pathlinker", "allpairs"]
+class TestParseOutputs:
+    def setup_class(cls):
+        """
+        Create the expected output directory
+        """
+        Path(OUTDIR).mkdir(parents=True, exist_ok=True)
+
+    def test_parse_outputs(self):
+        for algo in algorithms:
+            test_file = INDIR + f"{algo}-raw-pathway.txt"
+            out_file = OUTDIR + f"{algo}-pathway.txt"
+            print(out_file)
+            runner.parse_output(algo, test_file, out_file)
+            assert filecmp.cmp(OUTDIR +f"{algo}-pathway.txt", EXPDIR + f"{algo}-pathway-expected.txt")
+
+    def test_domino_parse_output(self):
```

Comment:
Does DOMINO still need a separate test case? It looks like it follows the same pattern as the others except it unlinks the output file before running.

If we merge it with the others, I would like to keep the comments somewhere that describe where the DOMINO input came from.